main.go

```diff
@@ -24,8 +32,25 @@ func main() {
 	utils.Assert("error creating discord session", err)
 
 	discord.AddHandler(coreMessageHandler)
+
+	user, err := discord.User("@me")              //grabbing account information
+	utils.Assert("error retrieving account", err) //check if error occurred
+	botID = user.ID                               //botID is a variable set to the bots information
+	botname = "LANFestBot"
+
+	gameLibFile, err := ioutil.ReadFile("games.yml")
+	var yamlErr error
+	if err != nil {
+		yamlErr = yaml.Unmarshal(gameLibFile, &gameLib)
+	}
+	if err != nil || yamlErr != nil {
+		fmt.Printf("%s %s Error loading game library config file. LFG and game matching features will not work.\n", err, yamlErr)
+	}
+
+	discord.AddHandler(commandHandler) // a listener that when it picks up a message create it runs the function
```

Comment:
Bad merge again.  The main message handler is named 'coreMessageHandler', and we add it as a handler on line 34.